src/coreclr/jit/lower.cpp

```diff
@@ -1793,6 +1799,39 @@ void Lowering::LowerCall(GenTree* node)
     // All runtime lookups are expected to be expanded in fgExpandRuntimeLookups
     assert(!call->IsExpRuntimeLookup());
 
+#ifdef TARGET_AMD64
+    if (call->gtCallMoreFlags & GTF_CALL_M_SPECIAL_INTRINSIC)
```

Comment:
Would be nice to fold this logic into a separate method e. g. `LowerCallMemmove`.